src/stories/MovieFormComposition/Composition.module.css

```diff
@@ -0,0 +1,11 @@
+.confirmationText {
```

Comment:
Maybe this file should be called `MovieFormComposition.module.css` or `DeleteMovie.stories.module.css`?